modules/users/client/controllers/admin/view-applicants.client.controller.js

```diff
@@ -0,0 +1,45 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('users.admin')
+    .controller('ViewApplicantsController', ViewApplicantsController);
+
+  ViewApplicantsController.$inject = ['$scope', '$filter', '$window', 'ApplicantsService', 'Notification'];
+
+
+  function ViewApplicantsController($scope, $filter, $window, ApplicantsService, Notification) {
+    var vm = this;
+    ApplicantsService
+      .query(function (data) {
+        vm.unapprovedUsers = data;
+      });
+
+
+    vm.removeApplicant = function (user) {
+      if ($window.confirm('Are you sure you want to delete this user?')) {
+        if (user) {
+          vm.unapprovedUsers.splice(vm.unapprovedUsers.indexOf(user), 1);
+          ApplicantsService.remove(user);
+          Notification.success('User deleted successfully!');
+        }
+      }
+    };
+
+    vm.approvalSwap = function (user) {
+      if (user) {
+          var newUser = user;
+          user.approvedStatus = true;
+          ApplicantsService.approvalSwap(newUser); //This database call isn't working.
+          vm.unapprovedUsers.splice(vm.unapprovedUsers.indexOf(user), 1);
```

Comment:
The logic to change a user's status should be implemented in the backend for both best practice and security reasons